protocol/0032-PRIM-price_monitoring.md

```diff
@@ -45,10 +45,15 @@ Likewise, pre-processing transactions will be needed as part of the [fees spec](
   - `probability` - probability level used in price monitoring. Must be in the (0,1) range.
   - `auctionExtension` - auction duration (or extension in case market is already in auction mode) per breach of the `horizon`, `probability` trigger pair specified above. Must be greater than 0.
 
+If any of the above parameters or the risk model gets modified in any way the price monitoring engine should be reset. If market is (price history gets cleared and bounds get recalculated). If the market is price monitoring auction (or auction extension triggered by price monitoring engine):
+
+- any remaining price bounds calculated prior to the update should get disactivated,
+- the auction end time implied by the currently running auction/extension should remain unchanged,
+- when auction uncrosses price monitoring should get reset using the updated parameters.
+
 ### Network
 
 - `market.monitor.price.defaultParameters`: Specifies default market parameters outlined in the previous paragraph. These will be used if market parameters don't get explicitly specified.
-- `market.monitor.price.updateFrequency`: Specifies how often (expressed in seconds) the price monitoring scaling factors should be updated by the risk model.
```

Comment:
This is no longer part of the implementation (superseded by the state variable engine and its update frequency parameter) so got rid of it as part of this PR.

Reply:
👍